packages/ns-api/README.md

```diff
@@ -1947,10 +1947,206 @@ api-cli ns.redirects list-zones
 ```
 
 Response example:
-```
+```json
 {
   "zones": [
     "guest"
   ]
 }
 ```
+
+## ns.dpi
+
+Manage netifyd DPI engine.
+
+### enable filtering
+
+To enable filtering, a boolean must be set in `dpi` configuration.
+The value can be set with:
+
+```bash
+uci add dpi.config.enabled='1'
+uci commit
+reload_config
+```
```

Comment:
This is not an API documentation ...
It's already documented inside ns-dpi package, no need to add it also here.